themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -98,6 +96,9 @@ JS;
           $this->headScript()->appendFile("covers.js");
       }
     ?>
+    <?=$this->cookieConsent()?>
+    <?=$this->headLink()?>
+    <?=$this->headStyle()?>
```

Comment:
That was done because the helper added css files which would have been missing, but now that the styles come from less files it's not needed anymore. Now reverted.